app/views/partials/header.ejs

```diff
@@ -68,8 +68,8 @@
             <% if(currentUser && currentUser.role === 'author') {  %>
                 <li><h3><a href="/articles/new">New Article</a></h3></li>
                 <li>
-                    <a href="/authors/<%=currentUser._id%>">
-                        <h3><a class="header__link" href="/authors/<%=currentUser._id%>">My account</a></h3>
+                    <a href="/authors/<%=currentUser.fullname%>">
```

Comment:
Does it already take into a count dashes?

Reply:
I think this needs to be currentUser.link, let me check it

Reply:
Fixed